src/Compat/FormDecorator/LabelDecorator.php

```diff
@@ -6,75 +6,60 @@
 use ipl\Html\Contract\Form;
 use ipl\Html\Contract\FormElement;
 use ipl\Html\Contract\FormDecoration;
+use ipl\Html\HtmlDocument;
 use ipl\Html\HtmlString;
 use ipl\Html\FormDecoration\LabelDecorator as IplHtmlLabelDecorator;
 use ipl\Html\ValidHtml;
 use ipl\Html\Attributes;
 use ipl\Html\HtmlElement;
-
+use ipl\I18n\Translation;
+use ipl\html\Text;
 
 class LabelDecorator extends IplHtmlLabelDecorator implements FormDecoration
 {
+    use Translation;
+
+    protected $requiredExplanationNeeded = false;
+
     /**
      * Decorates the label of the form element and adds a tooltip if it is required
      */
     protected function getElementLabel(FormElement $formElement): ?ValidHtml
     {
-        $elementAttrs = $formElement->getAttributes();
-        $label = $formElement->getLabel();
-
-        if (! $elementAttrs->has('id')) {
-            $elementAttrs->set('id', uniqid('form-element-'));
-        }
-
-        $result = new HtmlElement(
-            'label',
-            null,
-            HtmlString::create($label)
-        );
+        $result = parent::getElementLabel($formElement) ?? HtmlString::create('&nbsp;');
```

Comment:
Not sure if it's a good idea to render something like that: `&nbsp; <span…`

Please return early with the non-breaking space.